### PR TITLE
Threshold flow/v4

### PR DESCRIPTION
**Pull request by victorjulien**
SV_BRANCH=https://github.com/OISF/suricata-verify/pull/1679

replaces https://github.com/OISF/suricata/pull/10568:
- rebase
- reorder keyword id's a bit more
- fix nasty content limit flag abuse/mistake

### CODE REVIEW

**Review by inashivb — COMMENTED** [review]
Makes sense to me. Looks good. 👍🏽

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 18984

File: src/util-threshold-config.c [hunk 1]
```diff
@@ -796,6 +800,8 @@ static int ParseThresholdRule(const DetectEngineCtx *de_ctx, char *rawstr, uint3
             }
             else if (strcasecmp(th_track,"by_rule") == 0)
                 parsed_track = TRACK_RULE;
+            else if (strcasecmp(th_track, "by_flow") == 0)
```
Comment (jasonish) [inline src/util-threshold-config.c:803]: Should have an associated doc update.

**Comment by victorjulien** [comment]
replaced by #10595

**Pull request closed without merging**